### PR TITLE
glib::Binding improvements

### PR DESCRIPTION
**Pull request by sdroege**
Fixes some bugs and makes it automatically thread-safe once https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1745 is in and the user is using GLib 2.68 or newer.

### CODE REVIEW

File: glib/Gir_GObject.toml [hunk 1]
```diff
@@ -29,16 +29,24 @@ name = "GObject.Binding"
 status = "generate"
 trait = false
 concurrency = "send+sync"
-       # Fixed in GLib 2.66.2
        [[object.function]]
        name = "get_source"
-           [object.function.return]
-           nullable = true
-       # Fixed in GLib 2.66.2
+       # Not-threadsafe, see https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1745
```
Comment (GuillaumeGomez) [inline glib/Gir_GObject.toml:34]:
```suggestion
       # Not thread-safe, see https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1745
```

File: glib/Gir_GObject.toml [hunk 2]
```diff
@@ -29,16 +29,24 @@ name = "GObject.Binding"
...
+       # Using the GObject properties will be automatically safe with GLib 2.68.
+       ignore = true
        [[object.function]]
        name = "get_target"
-           [object.function.return]
-           nullable = true
+       # Not-threadsafe, see https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1745
```
Comment (GuillaumeGomez) [inline glib/Gir_GObject.toml:39]:
```suggestion
       # Not thread-safe, see https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1745
```

File: glib/Gir_GObject.toml (hunk 1 above)
Comment (sdroege) [reply glib/Gir_GObject.toml]: Thanks, fixed both

File: glib/src/gobject/binding.rs [hunk 3]
```diff
@@ -0,0 +1,38 @@
+// Copyright 2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <https://opensource.org/licenses/MIT>
+use crate::translate::*;
```
Comment (GuillaumeGomez) [inline glib/src/gobject/binding.rs:4]: Please add an empty line after the license comment. :)

**Review by GuillaumeGomez — APPROVED** [review]

File: glib/src/gobject/binding.rs (hunk 3 above)
Comment (sdroege) [reply glib/src/gobject/binding.rs:5]: Done

**Comment by GuillaumeGomez** [comment]
Then it's ready to go. Thanks!

**Pull request merged**